WordPress/src/main/res/layout/plan_details_fragment.xml

```diff
@@ -1,64 +1,68 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<org.wordpress.android.widgets.WPScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<org.wordpress.android.widgets.WPScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/scroll_view"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="match_parent"
+    android:background="@color/white">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
         android:layout_marginEnd="@dimen/margin_large"
-        android:paddingStart="@dimen/content_margin"
+        android:layout_marginStart="@dimen/margin_large"
+        android:orientation="vertical"
         android:paddingEnd="@dimen/content_margin"
-        android:layout_marginStart="@dimen/margin_large">
+        android:paddingStart="@dimen/content_margin"
+        android:paddingTop="@dimen/margin_extra_large">
 
         <ImageView
             android:id="@+id/image_plan_icon"
             android:layout_width="@dimen/plan_icon_size"
             android:layout_height="@dimen/plan_icon_size"
             android:layout_gravity="center_horizontal"
-            android:layout_marginTop="@dimen/margin_extra_large"
-            tools:background="@drawable/penandink_223dp"
-            android:contentDescription="@null"/>
+            android:contentDescription="@null"
+            android:src="@drawable/ic_plans_white_24dp"
+            android:tint="@color/grey"/>
```

Comment:
```suggestion
            tools:tint="@color/grey"/>
```

The `android:tint` attribute causes the plan icon to be `grey` (`#87a6bc`) even after it has been loaded form the icon URL.  See the screenshot below for illustration.

<img src="https://user-images.githubusercontent.com/3827611/52676004-d9531500-2ee5-11e9-8541-bad230e07f3d.png" width="240">

We should change the namespace to `tools`.